package.json

```diff
@@ -44,10 +44,11 @@
     "aframe": "github:mozillareality/aframe#hubs/master",
     "aframe-billboard-component": "^2.0.0",
     "aframe-inspector": "^0.8.3",
-    "aframe-physics-extras": "github:mozillareality/aframe-physics-extras#bugfix/physics-collider-world",
-    "aframe-physics-system": "github:mozillareality/aframe-physics-system#93b1dfe64c00587a9d09e1a8071750e75f340815",
+    "aframe-motion-capture-components": "github:mozillareality/aframe-motion-capture-components#aframe090",
```

Comment:
I removed motion-capture-components intentionally. Do we need it again?

Reply:
bad merge, thanks